cmd/cagent/main.go

```diff
@@ -108,6 +109,16 @@ func main() {
 		return
 	}
 
+	if *testConfigPtr {
+		err := ca.TestHub()
+		if err != nil {
+			fmt.Printf("Cagent HUB test failed: %s\n", err.Error())
```

Comment:
panic in error case or return a proper exit code?

Reply:
the problem is that `cagent -t` called in the `dpkg install` postscript. If I use the non-zero return code the installation will possibly fail. I will test this now